src/locales/en.json

```diff
@@ -1,4 +1,17 @@
 {
   "styleguide": "Styleguide",
-  "home": "Home page"
+  "home": "Home page",
+  "First Name": "First Name",
+  "Last Name": "Last Name",
+  "Email": "Email",
+  "Password": "Password",
+  "Password confirmation": "Password confirmation",
+  "password": "password",
+  "password confirmation": "password confirmation",
+  "BaseInput example": "BaseInput component (email)",
+  "emailPH": "john_kennedy@binarcode.com",
+  "emailLabel": "Email: ",
+  "Submit": "Submit",
+  "forgotPassText": "Forgot your password?",
+  "Sign Up": "Sign up"
```

Comment:
Let's change the keys:
`{
  "general.styleguide.tile": "Styleguide",
  "general.styleguide.section.baseInput": "BaseInput component (email)",
  "general.homePage.title": "Home page",
  
  
  "general.submit": "Submit",
  
  "register.label.firstName": "First Name",
  "register.label.lastName": "Last Name",
  "register.name.email": "Email",
  "register.label.email": "Email:",
  "register.label.password": "Password",
  "register.label.passwordConfirmation": "Password confirmation",
  "register.button.label.signUp": "Sign up",
  "register.placeholder.email": "john_kennedy@binarcode.com",
}`